lms/templates/courseware/progress.html

```diff
@@ -28,6 +28,9 @@
 <script type="text/javascript" src="${static.url('js/courseware/credit_progress.js')}"></script>
 <script>
   ${progress_graph.body(grade_summary, course.grade_cutoffs, "grade-detail-graph", not course.no_grade, not course.no_grade)}
+  function redirect_to_credit_help() {
```

Comment:
Please use jquery `click` method instead of javascript `onClick` functionality: http://stackoverflow.com/questions/12627443/jquery-click-vs-onclick